packages/teacher-app/public/firebase-messaging-sw.js

```diff
@@ -9,14 +9,14 @@ importScripts(
   "https://www.gstatic.com/firebasejs/9.1.0/firebase-messaging-compat.js"
 );
 // Initialize the Firebase app in the service worker by passing the generated config
+
 const firebaseConfig = {
-  apiKey: "AIzaSyCnIWbUvAk2dhkxDGZiaTnU95JXYQX-G7g",
-  authDomain: "shiksha-d8cb0.firebaseapp.com",
-  projectId: "shiksha-d8cb0",
-  storageBucket: "shiksha-d8cb0.appspot.com",
-  messagingSenderId: "941486633409",
-  appId: "1:941486633409:web:c5183689f3e0ab2e90bf40",
-  measurementId: "G-S69RMSE553",
+  apiKey: "AIzaSyDW2E975sqlALwHA2IwAITVCp95L8N2aS4",
+  authDomain: "demoparesh-3ac16.firebaseapp.com",
```

Comment:
Hey @sagarkoshti1990 why we make firebaseConfig different constant in every module can't we import it directly from common-lib?

Reply:
I was try but cant import